Tasks/GooglePlayReleaseV4/modules/metadataHelper.ts

```diff
@@ -61,27 +61,50 @@ async function addAllReleaseNotes(versionCodes: number[], languageCode: string,
     }
 
     const releaseNotes: pub3.Schema$LocalizedText[] = [];
+    let fullDefaultChangelogPath: string = '';
     for (const changelogFile of changelogs) {
         const changelogName: string = path.basename(changelogFile, path.extname(changelogFile));
         const changelogVersion: number = parseInt(changelogName, 10);
+        if (changelogName === 'default') {
+            fullDefaultChangelogPath = path.join(changelogDir, changelogFile);
+            continue;
+        }
+
         if (!isNaN(changelogVersion) && (versionCodes.indexOf(changelogVersion) !== -1)) {
             const fullChangelogPath: string = path.join(changelogDir, changelogFile);
-
-            console.log(tl.loc('AppendChangelog', fullChangelogPath));
-            releaseNotes.push({
-                language: languageCode,
-                text: getChangelog(fullChangelogPath)
-            });
-            tl.debug(`Found release notes version ${changelogVersion} from ${fullChangelogPath} for language code ${languageCode}`);
+            const message: string = `Found release notes version ${changelogVersion} from ${fullChangelogPath} for language code ${languageCode}`;
+            appendChangelogToReleaseNotes(fullChangelogPath, releaseNotes, languageCode, message);
         } else {
             tl.debug(`The name of the file ${changelogFile} is not a valid version code. Skipping it.`);
         }
     }
 
+    if (releaseNotes.length === 0 && fullDefaultChangelogPath !== '') {
+        const message: string = `'default.txt' release notes file is found. It is used for language code ${languageCode}`;
```

Comment:
Why it should be required?

Reply:
Sorry, see it, thanks! It will be used only if there are no other changelogs as default - so it's optional in general